sdk/src/main/java/co/omise/android/models/Capability.kt

```diff
@@ -0,0 +1,62 @@
+package co.omise.android.models
+
+import android.os.Parcel
+import android.os.Parcelable
+import co.omise.android.api.Endpoint
+import co.omise.android.api.RequestBuilder
+import okhttp3.HttpUrl
+import org.json.JSONObject
+
+/**
+ * Represents Capabilities object and contains its {@link RequestBuilder}.
+ *
+ * @see <a href="https://www.omise.co/capability-api">Capabilities API</a>
+ */
+class Capability : Model {
+    @JvmField
+    var banks: List<String>? = null
+    @JvmField
+    var paymentMethods: List<PaymentMethod>? = null
+    @JvmField
+    var zeroInterestInstallments: Boolean = false
+
+
+    constructor(rawJson: String) : this(JSONObject(rawJson))
+
+    private constructor(json: JSONObject) : super(json) {
+        banks = JSON.stringList(json, "banks")
+        paymentMethods = JSON.modelList(json, "payment_methods", PaymentMethod::class.java)
+        zeroInterestInstallments = JSON.bool(json, "zero_interest_installments")
+    }
+
+    /**
+     * The {@link RequestBuilder} class for retrieving account Capabilities.
+     */
+    class GetCapabilitiesRequestBuilder : RequestBuilder<Capability>() {
+
+        override fun path(): HttpUrl = buildUrl(Endpoint.API, "capability")
+
+        override fun type(): Class<Capability> = Capability::class.java
+    }
+
+    override fun writeToParcel(parcel: Parcel, flags: Int) {
```

Comment:
What about using **Kotlin Serializable** instead of manually build Pracel?

Reply:
We can also do this as part of the Serialization task. But does if we use `Kotlin Serializable`, do Java people also get serializability?

Reply:
Yes, it's support.

Reply:
okk then we'll implement it on all models in the Serialization PR 👍